### PR TITLE
Added "Account Settings" tab to Preferences.

### PR DESCRIPTION
**Pull request by jerdzica**

### CODE REVIEW

File: addons/bestja_volunteer/views/volunteer.xml [hunk 1]
```diff
@@ -107,6 +107,14 @@
                                     <field name="daypart_comments" placeholder="Wpisz dodatkowe informacje odnośnie Twojej dostępności czasowej lub pozostaw pole puste."/>
                                 </group>
                             </page>
+                            <page string="Ustawienia konta" attrs="{'invisible': [('user_access_level', 'not in', ['owner'])]}">
+                                <separator string="Zmiana hasła"/>
+                                <div class="oe_edit_only oe_grey">Zmienisz je w kolejnym kroku</div>
+                                    <button name="preference_change_password" type="object" string="zmień hasło" class="oe_link"/>
```
Comment (KrzysiekJ) [inline addons/bestja_volunteer/views/volunteer.xml]: Wrong indentation.

File: addons/bestja_volunteer/models.py [hunk 2]
```diff
@@ -50,8 +50,14 @@ class Volunteer(models.Model):
...
         'message_template.mixin'
-    ] 
+    ]
 
+    EMAIL_NOTIFICATIONS = [
+        ('always', 'Tak, chcę otrzymywać wszystkie powiadomienia'),
+        ('none', 'Nie chcę otrzymywać żadnych powiadomień'),
+    ]
+
+    notify_email = fields.Selection(EMAIL_NOTIFICATIONS, help='')
```
Comment (ludwiktrammer) [inline addons/bestja_volunteer/models.py]: `help=None` instead of an empty string.

**Comment by ludwiktrammer** [comment]
:+1:

**Comment by KrzysiekJ** [comment]
Merged in c61faaa.

**Pull request closed without merging**